### PR TITLE
Add examples for Card

### PR DESCRIPTION
**Pull request by jaime-lynn**
#35 
Feed is done currently with base semantic-ui. A separate issue will be to update that once the feed component is completed.

### CODE REVIEW

**Comment by jaime-lynn** [comment]
Everything works on these except for the ImageCard2 example. I'm having trouble getting the dimmer event to trigger. I'll keep working on it, but if you have any ideas, I'd love to hear them!

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Installing Semantic UI Vue provides the JavaScript for your components. You'll a
 Once Semantic UI Vue has been installed, you will need to import it in your main file (usually *index.js* or *main.js*) and tell Vue to use it:
 
 ```js
-import Vue form 'vue';
+import Vue from 'vue';
```
Comment (mariolamacchia) [inline README.md]: Oops, thanks!

File: docs/examples/views/CardExample/index.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+import RaisedCard from './RaisedCard.example';
+import RaisedLinkCard from './RaisedLinkCard.example';
+import LinkCard1 from './LinkCard1.example';
+import LinkCard2 from './LinkCard2.example';
+import FloatedContentCard from './FloatedContentCard.example';
+import TextAlignmentCard from './TextAlignmentCard.example';
+import ColoredCard from './ColoredCard.example';
+import ColumnCountCard from './ColumnCountCard.example';
+import StackableCard from './StackableCard.example';
+import DoublingCard from './DoublingCard.example';
```
Comment (mariolamacchia) [inline docs/examples/views/CardExample/index.js]: Can we order the imports alphabetically?

**Comment by mariolamacchia** [comment]
> Everything works on these except for the ImageCard2 example. I'm having trouble getting the dimmer event to trigger. I'll keep working on it, but if you have any ideas, I'd love to hear them!

Sorry, I missed the comment, but it seems you found the solution... great!

File: docs/examples/views/CardExample/index.js (hunk 2 above)
Comment (jaime-lynn) [reply docs/examples/views/CardExample/index.js]: Yes, absolutely! I'll get an update up for that shortly.

**Comment by mariolamacchia** [comment]
Released in v0.0.26

**Pull request merged**